.github/workflows/js-artifacts.yml

```diff
@@ -29,6 +29,11 @@ jobs:
             package_type: 'macOS-arm64'
             runner_labels: '["macos-15", "self-hosted"]'
 
+          - os_name: 'Linux'
+            arch: 'aarch64'
```

Comment:
Why do we have "arm64" for macOS and "aarch64" for Linux? Should we align that?

Reply:
There are references in setup which expect `arm64` not `aarch64` even on Linux: https://github.com/LadybirdBrowser/ladybird/blob/master/.github/actions/setup/action.yml#L122-L127

Reply:
Should we align it? Maybe? From what I understand, the value of CMAKE_SYSTEM_PROCESSOR when not cross-compiling comes from CMAKE_HOST_SYSTEM_PROCESSOR, which comes from `uname -m`.

Reply:
If not aligned, shouldn't the lines I linked in setup be updated too since it would never run currently since it is checking for `arm64` instead of `aarch64`?

Reply:
(I checked on my machine and `CMAKE_SYSTEM_PROCESSOR` is `aarch64` on Linux)

Reply:
> If not aligned, shouldn't the lines I linked in setup be updated too since it would never run currently since it is checking for arm64 instead of aarch64?

so much for my 4-line diff. guess I'll have to actually test this on my fork :grimacing: